### PR TITLE
Add Coingecko

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

File: entries/c/coingecko.com.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "Coingecko": {
+    "domain": "coingecko.com",
+    "contact": {
+      "facebook": "CoinGecko",
```
Comment (hkamran80) [inline entries/c/coingecko.com.json]:
```suggestion
```

Facebook handles haven't been working

File: entries/c/coingecko.com.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+      "twitter": "coingecko"
+    },
+    "categories": ["cryptocurrencies"]
```
Comment (hkamran80) [inline entries/c/coingecko.com.json]:
```suggestion
    "categories": [
      "cryptocurrencies"
    ]
```

File: img/c/coingecko.com.svg [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	c3.8,0,7.4,1.5,10,4.1C106.8,291.8,108.3,295.4,108.3,299.2L108.3,299.2z M108.3,299.2"/>
+<path class="st6" d="M104.1,299.3c0,4-2.4,7.7-6.2,9.2c-3.7,1.5-8,0.7-10.9-2.2c-2.9-2.9-3.7-7.1-2.2-10.9c1.5-3.7,5.2-6.2,9.2-6.2
+	C99.6,289.3,104.1,293.8,104.1,299.3L104.1,299.3z M104.1,299.3"/>
+<path class="st4" d="M159.5,319.6c-11.5,8.1-24.6,14.2-43.1,14.2c-8.7,0-10.4-9.2-16.2-4.7c-3,2.3-13.4,7.5-21.7,7.2
+	c-8.3-0.4-21.7-5.3-25.4-22.9c-1.5,17.7-2.2,30.7-8.9,45.6c13.2,21.2,44.7,37.5,73.3,30.5c-3.1-21.5,15.7-42.5,26.2-53.2
+	C147.8,332.1,155.5,325.5,159.5,319.6L159.5,319.6z M159.5,319.6"/>
+<path class="st6" d="M159,320.1c-3.6,3.3-7.8,5.7-12.1,7.8c-4.4,2.1-8.9,3.7-13.6,5c-4.7,1.2-9.6,2.1-14.5,1.7
+	c-5-0.4-10-2.1-13.3-5.8l0.2-0.2c4,2.6,8.7,3.5,13.3,3.7c4.7,0.1,9.4-0.2,14-1.1c4.6-1,9.2-2.4,13.6-4.2c4.4-1.8,8.7-3.9,12.5-6.9
+	L159,320.1z M159,320.1"/>
+</svg>
```
Comment (hkamran80) [inline img/c/coingecko.com.svg]: Here's a minified SVG

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 158.58 158.68"><path fill="#8dc63f" d="M158.58 79a79.3 79.3 0 0 1-135.1 56.7 79.44 79.44 0 0 1-17.6-86.4c12.1-29.7 41-49.1 73-49.3 43.9-.1 79.5 35.2 79.7 79Zm0 0"/><path fill="#f9e988" d="M152.68 79a73.4 73.4 0 0 1-125 52.5A73.3 73.3 0 0 1 78.88 6.1c40.6-.2 73.6 32.4 73.8 72.9zm0 0"/><path fill="#fff" d="M80.68 10.8c4.7-.8 9.4-.8 14.1 0 4.7.7 9.3 2.3 13.5 4.5 4.2 2.3 7.8 5.4 11.4 8.3 3.6 2.9 7.2 5.9 10.6 9.2 3.5 3.2 6.5 6.9 9 10.9 2.6 4 4.7 8.3 6.3 12.7 3.1 9 4.1 18.7 2.6 27.9h-.5c-1.5-9.2-3.6-17.9-6.9-26.2-1.5-4.2-3.4-8.3-5.4-12.2-2.2-3.9-4.5-7.8-7.1-11.4-2.6-3.7-5.9-7-9.6-9.7-3.7-2.7-8-4.4-12.1-6-4.1-1.6-8.3-3.3-12.6-4.6-4.3-1.2-8.7-2.2-13.3-3.1zm0 0"/><path fill="#8bc53f" d="M116.48 53.1c-5.3-1.5-10.8-3.7-16.4-5.9-.3-1.4-1.6-3.1-4.1-5.3-3.6-3.2-10.5-3.1-16.4-1.7-6.5-1.5-13-2.1-19.2-.6-50.7 14-22 48-40.6 82.3 2.6 5.6 31.2 38.4 72.5 29.6 0 0-14.1-33.9 17.8-50.2 25.9-13.2 44.5-37.8 6.4-48.2zm0 0"/><path fill="#009345" d="M79.58 40.2c3.7.3 17.1 4.6 20.5 7-2.9-8.3-12.5-9.4-20.5-7zm0 0"/><path fill="#fff" d="M122.78 75.4c0 2.7-3.3 4.1-5.2 2.2-1.9-1.9-.6-5.2 2.1-5.3 1.7.1 3.1 1.4 3.1 3.1zm-39.7-14.1c0 5.7-3.5 10.9-8.8 13.1-5.3 2.2-11.4 1-15.5-3.1s-5.3-10.2-3.1-15.5c2.2-5.3 7.4-8.8 13.1-8.8a14.31 14.31 0 0 1 14.3 14.3zm0 0"/><path fill="#58595b" d="M78.88 61.4c0 4-2.4 7.7-6.2 9.2a10.06 10.06 0 0 1-13.1-13.1c1.5-3.7 5.2-6.2 9.2-6.2 5.6.1 10.1 4.6 10.1 10.1zm0 0"/><path fill="#8bc53f" d="M134.28 81.7c-11.5 8.1-24.6 14.2-43.1 14.2-8.7 0-10.4-9.2-16.2-4.7-3 2.3-13.4 7.5-21.7 7.2-8.3-.4-21.7-5.3-25.4-22.9-1.5 17.7-2.2 30.7-8.9 45.6 13.2 21.2 44.7 37.5 73.3 30.5-3.1-21.5 15.7-42.5 26.2-53.2 4.1-4.2 11.8-10.8 15.8-16.7zm0 0"/><path fill="#58595b" d="M133.78 82.2c-3.6 3.3-7.8 5.7-12.1 7.8-4.4 2.1-8.9 3.7-13.6 5-4.7 1.2-9.6 2.1-14.5 1.7-5-.4-10-2.1-13.3-5.8l.2-.2c4 2.6 8.7 3.5 13.3 3.7 4.7.1 9.4-.2 14-1.1 4.6-1 9.2-2.4 13.6-4.2 4.4-1.8 8.7-3.9 12.5-6.9zm0 0"/></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]
Please place Expo in a separate PR

**Comment by soodaayush** [comment]
Made some changes to the code.

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
CircleCI currently has a partial outage that's affecting this job.

**Comment by soodaayush** [comment]
CircleCI seems to be up now.

**Pull request merged**